### ISSUE TITLE
Check if any API endpoints have zero usage

### ISSUE THREAD

**Issue by gmelodie** [application-research/estuary#521]
might be good to be opinionated about API usage patterns. so if estuary-www doesn't need it and we realize it's not being called elsewhere, we should probably remove the endpoint

[source](https://github.com/application-research/estuary/issues/424#issuecomment-1312067500)

**Comment by alvin-reyes** [comment]
yeah this is why I thought you want to have the tracer. We can add a tracer on the middleware of echo and log all request for each endpoints. We can then save that to a DB to record the hits and follow (trace) the usage.

**Comment by en0ma** [comment]
@alvin-reyes I don't think this is the kind of answer a trace should give, it's an overkill. @anjor the api in prod has api logs turned off by default
